src/cookbook/use-multi-signature-transactions.md

```diff
@@ -0,0 +1,261 @@
+    isi
+})?;
+```
+
+```ts [TypeScript]
+declare const adminClient: Client
+declare const torii: ToriiRequirementsForApiHttp
+
+const keyPair1 = crypto.KeyPair.generate()
+const keyPair2 = crypto.KeyPair.generate()
+
+const accountId = sugar.accountId('mad_hatter', 'wonderland')
+const assetDefinitionId = sugar.assetDefinitionId('camomile', 'wonderland')
+
+const registerAccount = sugar.instruction.register(
+  sugar.identifiable.newAccount(accountId, [
+    freeScope(() => keyPair1.publicKey().toDataModel()),
+  ]),
+)
+const setSignatureCondition = sugar.instruction.mint(
+  datamodel.Value(
+    'SignatureCheckCondition',
+    datamodel.SignatureCheckCondition(
+      'AllAccountSignaturesAnd',
+      datamodel.VecPublicKey([
+        freeScope(() => keyPair2.publicKey().toDataModel()),
+      ]),
+    ),
+  ),
+  datamodel.IdBox('AccountId', accountId),
+)
+const registerAssetDefinition = sugar.instruction.register(
+  sugar.identifiable.newAssetDefinition(
+    assetDefinitionId,
+    datamodel.AssetValueType('Quantity'),
+    { mintable: datamodel.Mintable('Infinitely') },
+  ),
+)
+
+await adminClient.submitExecutable(
+  torii,
+  sugar.executable.instructions([
+    registerAccount,
+    setSignatureCondition,
+    registerAssetDefinition,
+  ]),
+)
+```
+
+:::
+
+2. Submit the transaction signed by the main account key, minting the
+   asset:
+
+::: code-group
+
+```rust [Rust]
+let mut mad_hatter_client = admin_client.clone();
+mad_hatter_client.key_pair = key_pair_1;
+mad_hatter_client.account_id = account_id.clone();
+
+let quantity: u32 = 42;
+let asset_id = AssetId::new(asset_definition_id, account_id.clone());
+let mint_asset = MintExpr::new(quantity.to_value(), IdBox::AssetId(asset_id.clone()));
+
+let transaction_1 = {
+    let tx =
+        mad_hatter_client.build_transaction([mint_asset.clone()], UnlimitedMetadata::new())?;
+    mad_hatter_client.sign_transaction(tx)?
+};
+mad_hatter_client.submit_transaction(&transaction_1)?;
+```
+
+```ts [TypeScript]
+const madHatterClient = new Client({
+  signer: new Signer(accountId, keyPair1),
+})
+
+const quantity = 42
+const assetId = sugar.assetId(accountId, assetDefinitionId)
+const mintAsset = sugar.instruction.mint(
+  sugar.value.numericU32(quantity),
+  datamodel.IdBox('AssetId', assetId),
+)
+
+const transaction = makeSignedTransaction(
+  makeTransactionPayload({
+    executable: sugar.executable.instructions(mintAsset),
+    accountId,
+  }),
+  madHatterClient.signer,
+)
+
+await Torii.submit(torii, transaction)
+```
+
+:::
+
+3. Let's query the asset to ensure that the transaction was not committed
+   yet, without the second signature:
+
+::: code-group
+
+```rust [Rust]
+let error = mad_hatter_client
+    .request(client::asset::by_id(asset_id.clone()))
+    .expect_err("Asset should not be found");
+
+assert!(matches!(
+    error,
+    ClientQueryError::Validation(ValidationFail::QueryFailed(QueryExecutionFail::Find(
+        FindError::Asset(_)
+    )))
+));
+```
```

Comment:
What if this account has already that asset? Is there any reliable way to check? Is there any way to store partially signed transaction?